index.yaml

```diff
@@ -1568,7 +1600,42 @@ entries:
       catalog.rancher.io/release-name: rancher-kiali-server
     apiVersion: v2
     appVersion: v1.32.0
-    created: "2021-05-12T13:30:34.869057-07:00"
+    created: "2021-06-24T18:29:16.169082-07:00"
+    description: Kiali is an open source project for service mesh observability, refer
+      to https://www.kiali.io for details. This is installed as sub-chart with customized
+      values in Rancher's Istio.
+    digest: 96dcad6cf5a61d1cb163f81576cb39522a89b087361213a5a604282f1eada3b3
+    home: https://github.com/kiali/kiali
+    icon: https://raw.githubusercontent.com/kiali/kiali.io/master/themes/kiali/static/img/kiali_logo_masthead.png
+    keywords:
+    - istio
+    - kiali
+    - networking
+    - infrastructure
+    maintainers:
+    - email: kiali-users@googlegroups.com
+      name: Kiali
+      url: https://kiali.io
+    name: rancher-kiali-server
+    sources:
+    - https://github.com/kiali/kiali
+    - https://github.com/kiali/kiali-ui
+    - https://github.com/kiali/kiali-operator
+    - https://github.com/kiali/helm-charts
+    urls:
+    - assets/rancher-kiali-server/rancher-kiali-server-1.32.100+up1.32.0.tgz
+    version: 1.32.100+up1.32.0
```

Comment:
Why is the version `1.32.100+up1.32.0` and not `1.32.0+up1.32.0`, as 0-99 are for upstream 0.